src/jllvm/materialization/ByteCodeCompileLayer.cpp

```diff
@@ -821,7 +842,7 @@ void codeGenBody(llvm::Function* function, const Code& code, const ClassFile& cl
             case OpCodes::Goto:
             {
                 auto target = consume<std::int16_t>(current);
-                builder.CreateBr(addBasicBlock(target + offset));
+                builder.CreateBr(basicBlocks.find(target + offset) -> second);
```

Comment:
You can write `basicBlocks[target + offset]`  instead of `basicBlocks.find(target + offset) -> second`, bit shorter and nicer and semantically the same in this specific case.

Ditto below.